packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -168,16 +164,29 @@ class AutoCompleteState extends State<AutoComplete> with AutoDisposeMixin {
 
     final ColorScheme colorScheme = Theme.of(context).colorScheme;
 
+    final autoCompleteTextStyle = Theme.of(context)
+        .regularTextStyle
+        .copyWith(color: colorScheme.autoCompleteTextColor);
+
+    final tileContents = searchAutoComplete.value == null
+        ? <TextSpan>[]
+        : searchAutoComplete.value
+            .map((text) => TextSpan(
+                  text: text,
+                  style: autoCompleteTextStyle,
+                ))
+            .toList();
+
+    const minTileHeight = 48.0;
```

Comment:
remove this and fix the ui to stop using tile.

Reply:
Done!